GameTracker-Agent/Utility/BackgroundTimer.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Threading;
+
+namespace GameTracker_Agent
+{
+    class BackgroundTimer
+    {
+        private readonly DispatcherTimer Timer;
+
+        public BackgroundTimer()
+        {
+            Timer = new DispatcherTimer
+            {
+                Interval = TimeSpan.FromMilliseconds(1000)
```

Comment:
Move `1000` to a readonly property.
IMHO scanning games each second is not needed. This interval could be larger